sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/SubExprEvaluationRuntimeSuite.scala

```diff
@@ -45,6 +45,8 @@ class SubExprEvaluationRuntimeSuite extends SparkFunSuite {
 
     val proxy3 = ExpressionProxy(Literal(3), 2, runtime)
     proxy3.eval()
+    // wait data Eviction
```

Comment:
@ben-manes I have another question, here I add `Thread.sleep(5L)` to wait data eviction, because I find that the data eviction has some delay

Reply:
You can disable async behavior by using `Caffeine.executor(Runnable::run)`. By default work is delegated to FJP.commonPool because we don't know how expensive the listeners might be. The cache itself is fast, so you can disable it for simplicity